hikari/errors.py

```diff
@@ -103,6 +104,17 @@ class HikariInterrupt(KeyboardInterrupt, HikariError):
     """The signal name that was raised."""
 
 
+@attr.define(auto_exc=True, repr=False, weakref_slot=False)
+class ComponentAlreadyRunningError(HikariError):
```

Comment:
Just merge this with ComponentNotRunningError and rename it to "ComponentStateConflictError"